cli/build.gradle

```diff
@@ -268,6 +269,12 @@ test {
 	maxParallelForks = forks
 }
 
+unzipManifest {
+	from zipTree("${buildDir}/libs/blade.jar")
```

Comment:
Can you use something like `from jar` which means from the output of the jar task?  That way unzipManifest will depend on jar properly so if someone does `gw unzipManifest` it will correctly build the jar.